app/src/main/java/com/example/tasky/core/presentation/designsystem/drop_downs/TaskyNotificationReminderDropdown.kt

```diff
@@ -0,0 +1,171 @@
+@file:OptIn(ExperimentalMaterial3Api::class)
+
+package com.example.tasky.core.presentation.designsystem.drop_downs
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.requiredWidth
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.filled.Check
+import androidx.compose.material.icons.filled.KeyboardArrowDown
+import androidx.compose.material3.DropdownMenu
+import androidx.compose.material3.DropdownMenuItem
+import androidx.compose.material3.ExperimentalMaterial3Api
+import androidx.compose.material3.Icon
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.MenuDefaults
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.rotate
+import androidx.compose.ui.layout.onSizeChanged
+import androidx.compose.ui.platform.LocalDensity
+import androidx.compose.ui.tooling.preview.PreviewLightDark
+import androidx.compose.ui.unit.DpOffset
+import androidx.compose.ui.unit.dp
+import com.example.tasky.agenda.presentation.util.AgendaDetailConfigProvider
+import com.example.tasky.agenda.presentation.util.AgendaReminderInterval
+import com.example.tasky.core.presentation.designsystem.theme.TaskyTheme
+import com.example.tasky.core.presentation.designsystem.theme.extended
+
+@Composable
+fun TaskyNotificationReminderDropdown(
```

Comment:
Minor: the current naming ties this component to “Reminder,” but since the UI is shared across all agenda items, it might be clearer to use a more general prefix like ...AgendaItem… (e.g. TaskyAgendaItemDropdownMenu). That way the name communicates its broader scope instead of sounding reminder-specific.